spec/support/helpers/model_spec_helpers.rb

```diff
@@ -1,8 +1,10 @@
 module ModelSpecHelpers
-  def expect_location_answer(answer, params)
+  def expect_location_answer(response, answer, params)
+    response.save
     answer.save
+
     # If the answer is invalid there is no need to check if values were copied correctly.
-    return if answer.invalid?
+    return if response.invalid?
```

Comment:
Also would like to go over what this means for validation flow.

Reply:
Yeah we should check into this.